pkg/scaffold/resource/resource.go

```diff
@@ -16,144 +16,42 @@ limitations under the License.
 
 package resource
 
-import (
-	"fmt"
-	"regexp"
-	"strings"
-
-	"github.com/gobuffalo/flect"
-)
-
 // Resource contains the information required to scaffold files for a resource.
 type Resource struct {
-	// Namespaced is true if the resource is namespaced
-	Namespaced bool
+	// Namespaced is true if the resource is namespaced.
+	Namespaced bool `json:"namespaces,omitempty"`
 
-	// Group is the API Group.  Does not contain the domain.
-	Group string
+	// Group is the API Group. Does not contain the domain.
+	Group string `json:"group,omitempty"`
 
-	// GroupImportSafe is the API Group.  Does not contain the domain and it the "-"
-	// It is used to do safe imports.
-	GroupImportSafe string
+	// GroupPackageName is the API Group cleaned to be used as the package name.
+	GroupPackageName string `json:"-"`
 
-	// Version is the API version - e.g. v1beta1
-	Version string
+	// Version is the API version.
```

Comment:
```suggestion
	// Version is the API version. - e.g. v1beta1
```

Reply:
I dont think this provides any value. As a kubebuilder user you will never see a code comment and as a developer you have some lines before the `versionPattern` which provides the possible value. Do you really think it helps?